ruby_challenges.rb

```diff
@@ -1,11 +1,23 @@
 # ASSESSMENT 5: Ruby Coding Practical Questions
 
 # --------------------1) Create a class called Bike that is initialized with a model, wheels, and a frame size. The default number of wheels is 2. Create a get_info method that returns a sentence with all the data about each bike object.
-
-
-
-
-
+class Bike 
+    def set_model model 
+        @model = model
+    end
+
+    def get_wheels
+        @wheels = "All bikes have 2 wheels"
```

Comment:
Think about the data type for wheels: string vs number and why this distinction is important